ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdteststun.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+import logging
+from functools import cached_property
+
+from ansible_collections.arista.avd.plugins.plugin_utils.eos_validate_state_utils.avdtestbase import AvdTestBase
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import get
+
+LOGGER = logging.getLogger(__name__)
+
+
+class AvdTestStun(AvdTestBase):
+    """
+    AvdTestStun class for STUN tests.
```

Comment:
Mention in docstring and doc that you test STUN only in the WAN case by looking into the path-groups.
STUN can be used for other things (though not in AVD today ;) )